### PR TITLE
Update serviceproviders.xml

### PR DESCRIPTION
**Pull request by atlochowski**
Update credentials for EE MMS in UK

### CODE REVIEW

**Comment by atlochowski** [comment]
https://community.ee.co.uk/t5/Mobile-Data/EE-APN-Settings-Where-to-find-them/td-p/1145404

File: mobile-broadband-provider-info/serviceproviders.xml [hunk 1]
```diff
@@ -4692,6 +4692,8 @@ conceived.
 			<apn value="everywhere">
 				<usage type="mms"/>
 				<name>MMS</name>
+				<username>eesecure</username>
+				<password>secure</password>
```
Comment (monich) [inline mobile-broadband-provider-info/serviceproviders.xml:4696]: This is so secure 🙂

**Review by monich — APPROVED** [review]

**Comment by monich** [comment]
Thanks!

File: mobile-broadband-provider-info/serviceproviders.xml (hunk 1 above)
Comment (atlochowski) [reply mobile-broadband-provider-info/serviceproviders.xml:4696]: I know, secure password in plain text :P

**Pull request merged**